packages/medusa/src/repositories/price-list.ts

```diff
@@ -8,16 +8,17 @@ import {
 } from "typeorm"
 import { PriceList } from "../models/price-list"
 import { CustomFindOptions, ExtendedFindConfig } from "../types/common"
+import { CustomerGroup } from "../models"
 
-type PriceListFindOptions = CustomFindOptions<PriceList, "status" | "type">
+export type PriceListFindOptions = CustomFindOptions<PriceList, "status" | "type">
 
 @EntityRepository(PriceList)
 export class PriceListRepository extends Repository<PriceList> {
   public async getFreeTextSearchResultsAndCount(
     q: string,
     options: PriceListFindOptions = { where: {} },
-    groups?: FindOperator<PriceList>,
-    relations: (keyof PriceList)[] = []
+    groups: FindOperator<(CustomerGroup | string)[]>,
```

Comment:
If it is a FindOperator of CustomerGroup, then should we map on the value to get only the ids?